katago-analyze-sgf-daemon.js

```diff
@@ -124,12 +125,25 @@ class Engine extends EventEmitter {
     if (this.katago && !this.katago.killed) {
       return
     }
-    this.katago = spawn(this.katagoPath, [
-      'analysis',
-      '-config',
-      this.analysisConfig,
-      '-quit-without-waiting',
-    ])
+    let katagoArgs
+    if (this.modelPath == null) {
+      katagoArgs = [
+        'analysis',
+        '-config',
+        this.analysisConfig,
+        '-quit-without-waiting',
+      ]
+    } else {
+      katagoArgs = [
+        'analysis',
+        '-config',
+        this.analysisConfig,
+        '-model',
+        this.modelPath,
+        '-quit-without-waiting',
+      ]
+    }
```

Comment:
There is still unnecessary code duplication here. Initialize the `katagoArgs` array once, and just append the `modelPath` if needed.